wordcloud/wordcloud.py

```diff
@@ -350,7 +350,7 @@ def process_text(self, text):
         d = {}
         flags = (re.UNICODE if sys.version < '3' and type(text) is unicode
                  else 0)
-        for word in re.findall(r"\w[\w']*", text, flags=flags):
+        for word in re.findall(r"\w[\w']+", text, flags=flags):
```

Comment:
actually, I'm not sure about this change any more. Maybe we should just customize the regexp as discussed in #109. Yours would be a better default, though.